spec/javascripts/models/beta/column_spec.js

```diff
@@ -0,0 +1,18 @@
+import * as Column from 'models/beta/column.js';
+import { status } from 'libs/beta/constants';
+
+describe('Column model', function() {
+  describe('isChillyBin', function() {
+    it('should return true when state is unscheduled', function() {
```

Comment:
I would create a `describe` here too.
Tips:
- `.toEqual(true)` === `.toBeTruthy()`
- `.toEqual(false)` === `.toBeFalsy()`
- Remove the `should` word because this makes the test look like is unsure.
> should: used to indicate what is probable.


```js
describe('When state is unscheduled', () => {
  const story = { state: status.UNSCHEDULED }
  
  it('returns truthy', () => {
    expect(Column.isChillyBin(story)).toBeTruthy();
  });
})

describe('When state is not unscheduled', () => {
  const story = { state: status.STARTED }
  
  it('returns falsy', function() {
    expect(Column.isChillyBin(story)).toBeFalsy();
  });
});
```